core/src/test/java/org/kohsuke/stapler/framework/io/LargeTextTest.java

```diff
@@ -53,13 +63,53 @@ public void writeLogTo() throws Exception {
             // right
         }
     }
+
     String tail(String text, long start) throws IOException {
-        ByteBuffer bb = new ByteBuffer();
-        bb.write(text.getBytes(), 0, text.length());
-        LargeText t = new LargeText(bb, true);
+        LargeText t;
+        try (ByteBuffer bb = new ByteBuffer()) {
+            bb.write(text.getBytes(), 0, text.length());
+
+            t = new LargeText(bb, true);
+        }
         ByteArrayOutputStream baos = new ByteArrayOutputStream();
         assertEquals(text.length(), t.writeLogTo(start, baos));
         return baos.toString();
     }
 
+    @Issue("#141")
+    @Test
```

Comment:
Great but please `@Ignore` it so we do not crash CI.

Reply:
@jglick done. @Ignore added